docs/review-version.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Submitting your paper for review
+---
+
+Below are the general *ACL policies for preparing the review version of your paper.
+
+If your question is not answered here, please email the current publications chairs for any questions or clarifications.
+We will update this page if new issues arise.
+
+## Outline
+{: .no_toc}
+
+- TOC
+{:toc}
+
+## Checklist
+
+- Ensure that your paper conforms to the [Formatting Guidelines](formatting.html).
+- Ensure that your paper is fully [anonymized](#anonymity).
+- Ensure that all [tables, graphs, and figures](formatting.html#figures-and-tables) are readable.
+- Ensure that the paper does not exceed the [page limit](formatting.html#paper-length).
+- Check the [paper size and embedded fonts](formatting.html#file-format).
+
+## Anonymity
+
+As reviewing will be double-blind, review versions must not include any identifying information about the authors (such as names, affiliations, or URLs).
+Self-references that reveal the author's identity, e.g.,
+
+> We previously showed (Gusfield, 1997)...
+
+must be avoided, and anonymous citations, e.g.,
+
+> We previously showed (Anonymous, 1997)...
+
+should also be avoided. Instead, use citations such as
+
+> Gusfield (1997) previously showed...
+
+Review versions must not include acknowledgements.
+
+**Papers that do not conform to these requirements may be rejected without review.**
+
+Any preliminary non-archival versions of submitted papers should be listed in the submission form but not in the review version of the paper.
+Reviewers are generally aware that authors may present preliminary versions of their work in other venues, but will not be provided the list of previous presentations from the submission form.
+
```

Comment:
This was a bit confusing to me. I think you are referring to the paper submission process, but whether this is possible varies by venue and venue policy.

Reply:
Hm, I agree it's confusing, but this policy comes from the existing formatting guidelines.

To unconfuse it, I suggest changing the first line to "should be listed in the submission form, but not in the paper itself."

Reply:
That's clearer.